modules/new-book.js

```diff
@@ -0,0 +1,11 @@
+const booksSection = document.querySelector('.display-books');
+const formSection = document.querySelector('.form-section');
+const contactSection = document.querySelector('.contact-section');
+
+function showForm() {
+  booksSection.classList.add('hide');
+  formSection.classList.remove('hide');
+  contactSection.classList.add('hide');
+}
```

Comment:
- [ ] Nice work there :clap: :sunglasses: But as requested by the project requirement, we should refactor the code using arrow function. Therefore let's change this `function` keyword and use arrow function.